script/update-aws-cdk

```diff
@@ -51,7 +55,7 @@ raisePR() {
 
   git checkout -b $BRANCH_NAME
 
-  for file in package.json package-lock.json integration-test/package.json; do
+  for file in package.json package-lock.json integration-test/package.json integration-test/package-lock.json; do
```

Comment:
https://github.com/guardian/cdk/pull/641#pullrequestreview-694701813

>  I think integration-test/package-lock.json should have changed too

Yep, this should fix it!